pkg/prometheus/common.go

```diff
@@ -57,6 +57,29 @@ var (
 	PrometheusK8sLabelName        = "app.kubernetes.io/name"
 	ProbeTimeoutSeconds     int32 = 3
 	LabelPrometheusName           = "prometheus-name"
+
+	reservedHeaders = map[string]struct{}{
```

Comment:
I don't feel comfortable maintaining a copy of the original map. One alternative would be to build a minimal headers map and call Validate() on it.  Another alternative is to ask making this map a public variable in prometheus/common.

Reply:
Thanks @simonpasquier 
I got it
I think it would be better to reference the Map variable in `prometheus/common`. ref: https://github.com/prometheus/common/pull/741

Before that, I would be to build a minimal headers map and call Validate() on it.

Reply:
@simonpasquier Please re-check this part. thanks 😄